appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectToolbar.java

```diff
@@ -35,10 +35,12 @@ interface ProjectToolbarUiBinder extends UiBinder<Toolbar, ProjectToolbar> {}
   private static final String WIDGET_NAME_SENDTONG = "Send to Gallery";
   private static final String WIDGET_NAME_LOGINTOGALLERY = "Login to Gallery";
 
-  private final boolean isReadOnly;
+  public final boolean isReadOnly;
+  public final boolean galleryEnabled;
+  @UiField public Label projectLabel;
+  @UiField public Label trashLabel;
```

Comment:
It seems like these should be `protected`, and the flags could even be `private` since they are only accessed from within this class.